apis/kueue/v1alpha2/clusterqueue_types.go

```diff
@@ -195,6 +196,7 @@ type Resource struct {
 	//
 	// +listType=map
 	// +listMapKey=name
+	// +kubebuilder:validation:MaxItems=16
```

Comment:
Should we also add MinItems=1 @alculquicondor , if we want to add resources, we should provide at least one flavor.

Reply:
Yes, we need set `MinItems=1`. It need at least one flavor.

Reply:
I think leaving `MakeResource` as-is is perfectly acceptable.